src/turnkeyml/version.py

```diff
@@ -1 +1 @@
-__version__ = "3.0.7"
+__version__ = "3.0.8"
```

Comment:
Should this be a patch or a Major?
https://semver.org/

Reply:
technically a major!

Reply:
updated.